### ISSUE TITLE
Automatic volume alerts for stocks in watchlist

### ISSUE THREAD

**Issue by TNowalk** [TNowalk/stock-slack-bot#7]
Send user DM when a stock has abnormally high volume in a users watchlist

Blocked by #6

**Comment by TNowalk** [comment]
I believe the best way to do this would be to get the average volume for a stock, divide it by 5 minute blocks to get the "average" per 5 minutes.  The script will run every 5 minutes, keep of volume since last poll, if it's some undetermined percent higher than the average then send alert.

Will probably want to implement some sort of caching to the previous volume as well as the calculated 5 minute average.

**Comment by shaunburdick** [comment]
Well it's node, so you can just use it's state-fulness. Create a class that, when started uses something like `setInterval()` to fire off a function every X minutes and you can keep the state inside the class object

**Comment by TNowalk** [comment]
True, there are "cron" like modules out there that basically just wrap around the interval but using setInterval is probably the cleanest solution.
